pkgs/development/libraries/opencv/3.x.nix

```diff
@@ -147,6 +149,6 @@ stdenv.mkDerivation rec {
     homepage = http://opencv.org/;
     license = stdenv.lib.licenses.bsd3;
     maintainers = with stdenv.lib.maintainers; [viric flosse mdaiter];
-    platforms = with stdenv.lib.platforms; linux;
+    platforms = with stdenv.lib.platforms; [ linux darwin ];
```

Comment:
I feel like we should reach a consensus on inherit vs using the darwin attributeset directly. Neither of them has a clear advantage, but I would prefer one of these to be used more prominently throughout nixpkgs.